Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md

```diff
@@ -38,9 +38,12 @@ Open powershell (in admin mode) and run the following commands. When prompted fo
 Requires the application instance ID created above. Continue your powershell session and run the following commands.
   * `New-CsTeamsComplianceRecordingPolicy -Enabled $true -Description "Test policy created by <yourName>" <policyIdentity>`
   * ```Set-CsTeamsComplianceRecordingPolicy -Identity <policyIdentity> -ComplianceRecordingApplications ` @(New-CsTeamsComplianceRecordingApplication -Parent <policyIdentity> -Id <objectId>)```
-
+New-CsTeamsComplianceRecordingApplication will create a application with Identiy name in format of Tag:<policyIdentity>/<objectId>.
+  * `Get-CsTeamsComplianceRecordingApplication -Identity Tag:<policyIdentity>/<objectId>` to verify the TeamsComplianceRecordingApplication created.
+  * `Set-CsTeamsComplianceRecordingApplication -Identity Tag:<policyIdentity>/<objectId> -RequiredBeforeMeetingJoin $false -RequiredDuringMeeting $false -RequiredBeforeCallEstablishment $false -RequiredDuringCall $false`
```

Comment:
Can we add a comment for this new cmd, what it does and why its required. 
Official docs for cmdlet are below
https://learn.microsoft.com/en-us/powershell/module/teams/set-csteamscompliancerecordingapplication?view=teams-ps